lib/prometheus_exporter/metric/histogram.rb

```diff
@@ -5,6 +5,16 @@ class Histogram < Base
 
     DEFAULT_BUCKETS = [0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5.0, 10.0].freeze
 
+    @default_buckets = nil if !defined?(@default_buckets)
+
+    def self.default_buckets
+      @default_buckets || DEFAULT_BUCKETS
+    end
+
+    def self.default_buckets=(buckets)
+      @default_buckets = buckets
+    end
+
     def initialize(name, help, opts = {})
       super(name, help)
       @buckets = (opts[:buckets] || DEFAULT_BUCKETS).sort.reverse
```

Comment:
how is this going to work if it is not used here? 

Is there precedence in the API for a default override like this?

Reply:
Good spot! I have just pushed changes in 31969581eadbec2c98c66177a4b72736f9aa3d16
Default buckets will be overwritten by this one specified in `opt[:buckets]` :+1:

Reply:
@SamSaffron Would you take a look? 🙏